core/src/scene/sceneLoader.cpp

```diff
@@ -450,6 +458,11 @@ void SceneLoader::loadStyleProps(Style& style, Node styleNode, Scene& scene) {
         else { LOGW("Invalid blend mode '%s'", str.c_str()); }
     }
 
+    if (Node blendOrderNode = styleNode["blend_order"]) {
+        auto blendOrder = blendOrderNode.as<int>();
```

Comment:
With new additions to the yaml parsing we should try to be robust to unexpected content (i.e. wrap conversions in a try-catch)